contentpacks/dubbed_video_mappings_submodule.py

```diff
@@ -0,0 +1,118 @@
+import errno
+import os
+import logging
+import json
+# from fle_utils.general import softload_json
+
+CACHE_VARS = []
+TOPICS_DATA_PATH = os.path.realpath(os.path.dirname(os.path.realpath(__file__)))
+TOPICS_FILEPATH = os.path.join(TOPICS_DATA_PATH, "resources", "topics.json")
+
+
+def softload_json(json_filepath, default={}, raises=False, logger=None, errmsg="Failed to read json file"):
+    if default == {}:
+        default = {}
+    try:
+        with open(json_filepath, "r") as fp:
+            return json.load(fp)
+    except Exception as e:
+        if logger:
+            logger("%s %s: %s" % (errmsg, json_filepath, e))
+        if raises:
+            raise
+        return default
+
+# Globals that can be filled
+TOPICS          = None
+CACHE_VARS.append("TOPICS")
```

Comment:
`CACHE_VARS` were a caching mechanism for the old ka-lite, and aren't needed here anymore.